Libraries/Image/RCTImageViewManager.m

```diff
@@ -49,11 +50,11 @@ - (UIView *)view
   view.renderingMode = json ? UIImageRenderingModeAlwaysTemplate : defaultView.renderingMode;
 }
 
-RCT_EXPORT_METHOD(getSize:(NSURLRequest *)request
+RCT_EXPORT_METHOD(getSize:(RCTImageSource *)source
```

Comment:
Should this be considered a breaking change?

Reply:
If someone is using this directly then it is, but I don't think you should be using it like that. The real exposed _documented_ API is defined in [Image.ios.js](https://github.com/facebook/react-native/pull/18850/files#diff-741217105b8bfdafc4248cbd8efa8eda) and is not changed in behaviour at all.

Reply:
@hramos